t/lib/t/MusicBrainz/Server/Controller/User/Edit.pm

```diff
@@ -57,13 +51,76 @@ $mech->get_ok($verify_email_path);
 $mech->content_contains("Thank you, your email address has now been verified!");
 
 $mech->get('/user/new_editor');
-$mech->content_contains('http://example.com/~new_editor/');
-$mech->content_contains('hello world!');
 $mech->content_contains('new_email@example.com');
 
 
 };
 
+test 'Limited users cannot edit website and biography' => sub {
```

Comment:
URL format yes, date checking no. Fixed in https://github.com/metabrainz/musicbrainz-server/pull/515/commits/cf46dea1e27697d92d3ea9201419c5c0f2bb165c